server/bin/pbench-unpack-tarballs.py

```diff
@@ -62,6 +63,11 @@ def unpack_tarball(
     tarlist = glob.iglob(os.path.join(config.ARCHIVE, "*", _linksrc, "*.tar.xz"))
     ntotal = nsuccess = 0
 
+    try:
+        file_tree = FileTree(config, logger)
+    except Exception:
+        logger.error("Unable to map the file tree")
```

Comment:
The `try`/`except` isn't necessary here since `FileTree.__init__` just initializes the instance attributes without doing anything that can fail.